_posts/2020-07-21-cube-functionality-to-kartothek.markdown

```diff
@@ -0,0 +1,414 @@
+---
+layout: single
+title:  "Introducing Cube Functionality To Kartothek"
+date:   2020-07-21 06:00:00 +0500
+tags: technology python data-engineering
+header:
+  overlay_image: assets/images/tech_gear_banner.jpg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Marco Neumann & Usha Nemani
+author_profile: true
+---
+# Introducing Cube Functionality To Kartothek
+
+In our last blog we introduced [Kartothek](2019-05-28-introducing-kartothek.markdown), a table management python library powered by Dask. 
+Our journey continued by adding a gem to our jewel. We empowered Kartothek with multiple-dataset functionality. 
+Kartothek already provides Dataset features, As a user do I really need a multiple-dataset? Hang On!! 
+Let us spend rest of our time understanding the story of **Cube (kartothek that supports multiple datasets)**. 
+
+## What is a Cube?
+A Cube deals with multiple Kartothek datasets.
+
+Let us start with building a cube for **geodata**. Similar to Kartothek, 
+we need a [simplekv](https://simplekv.readthedocs.io/)-based store backend along with an abstract cube definition.
```

Comment:
I think we should start with a short paragraph describing what we refer to as a cube and also separate our usage of the term from the more commonly used OLAP cube. Diving straight into the example might leave users confused / not knowing what to expect

Reply:
I would however not get to theoretical / deep-diving at this point, since I think an example-based explanation is easier to follow. (not saying that 1-2 sentences would be misplaced here)

Reply:
Added URL to `Data Cube`, in `what is a cube` section.